src/modules/launcher/Microsoft.Launcher/dllmain.cpp

```diff
@@ -294,10 +294,12 @@ class Microsoft_Launcher : public PowertoyModuleIface
         {
             if (WaitForSingleObject(m_hProcess, 0) == WAIT_OBJECT_0)
             {
+                Logger::warn("Launcher exited. Try to restart process.");
```

Comment:
Wording here is a bit confusing at first sight, at least for me.
What about "PowerToys Run has exited unexpectedly, Restarting PowerToys Run."?